src/main/resources/conf/config.yml

```diff
@@ -54,7 +53,7 @@ controllerInfo:
     tierName: ""
     nodeName: ""
 
-enableHealthChecks: true
+enableHealthChecks: false
```

Comment:
I think its better to remove this flag

Reply:
Made changes and removed enableHealthChecks flag from config file